src/test/analyze/attw.test.ts

```diff
@@ -0,0 +1,41 @@
+import {describe, it, expect} from 'vitest';
+import {runAttw} from '../../analyze/attw.js';
+import {LocalFileSystem} from '../../local-file-system.js';
+import * as path from 'node:path';
+
+describe('ATTW TypeScript Configuration', () => {
```

Comment:
nit: these are now just attw tests, not specifically typescript

(yes, right now these are testing the typescript checks, but its a test file for the attw integration in general)